File: 3weeks/First Work.c
#include <stdio.h>

int n, k, result;

int main(){
    scanf("%d %d", &n, &k);
    while(1){
        if((n%k)==0){
            n=n/k;
        }
        else{
            n=n-1;
        }
        result++;
        if(n<k) break;
    }
    result = result+(n-1);
    printf("%d", result);
    return 0;
}

File: 3weeks/First Work.py
n, k = raw_input().split()
result=0
n=int(n)
k=int(k)

while True:
    if n%k==0:
        n = n/k
    else:
        n = n -1
    
    result = result+1
    if n<k:
        break

result = result+(n-1)
print(result)

